auth0/src/test/java/com/auth0/android/myaccount/MyAccountAPIClientTest.kt

```diff
@@ -0,0 +1,243 @@
+package com.auth0.android.myaccount
+
+import com.auth0.android.Auth0
+import com.auth0.android.request.PublicKeyCredentials
+import com.auth0.android.request.Response
+import com.auth0.android.result.PasskeyAuthenticationMethod
+import com.auth0.android.result.PasskeyEnrollmentChallenge
+import com.auth0.android.util.AuthenticationAPIMockServer.Companion.SESSION_ID
+import com.auth0.android.util.MockMyAccountCallback
+import com.auth0.android.util.MyAccountAPIMockServer
+import com.auth0.android.util.SSLTestUtils.testClient
+import com.google.gson.Gson
+import com.google.gson.GsonBuilder
+import com.google.gson.reflect.TypeToken
+import com.nhaarman.mockitokotlin2.mock
+import okhttp3.mockwebserver.RecordedRequest
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.MockitoAnnotations
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+import java.util.Map
+
+
+@RunWith(RobolectricTestRunner::class)
+@Config(manifest = Config.NONE)
+public class MyAccountAPIClientTest {
+
+    private lateinit var client: MyAccountAPIClient
+    private lateinit var gson: Gson
+    private lateinit var mockAPI: MyAccountAPIMockServer
+
+    @Before
+    public fun setUp() {
+        mockAPI = MyAccountAPIMockServer()
+        MockitoAnnotations.openMocks(this)
+        gson = GsonBuilder().serializeNulls().create()
+        client = MyAccountAPIClient(auth0, ACCESS_TOKEN)
+    }
+
+    @After
+    public fun tearDown() {
+        mockAPI.shutdown()
+    }
+
+    @Test
+    public fun `passkeyEnrollmentChallenge should build correct URL`() {
+        val callback = MockMyAccountCallback<PasskeyEnrollmentChallenge>()
+        client.passkeyEnrollmentChallenge()
+            .start(callback)
+        val request = mockAPI.takeRequest()
+        assertThat(request.path, Matchers.equalTo("/me/v1/authentication-methods"))
+    }
+
+    @Test
+    public fun `passkeyEnrollmentChallenge should include correct parameters`() {
```

Comment:
Should we also include a test case that asserts that, by default, no params are being sent other than `"type": "passkey"`?